lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -33,23 +33,37 @@
 #include "EnumToString.h"
 #include "SessionStateEnforcer.h"
 
+std::shared_ptr<magma::SessionStateEnforcer> conv_session_enforcer;
 namespace magma {
-// temp routine
-void call_back_void_upf(grpc::Status, magma::UpfRes response) {
-  // do nothinf but to only passing call back
+void call_back_upf(grpc::Status, magma::UPFSessionContextState response) {
+  std::string imsi             = response.session_snaphot().subscriber_id();
+  uint32_t version             = response.session_snaphot().session_version();
+  uint32_t fteid               = response.session_snaphot().local_f_teid();
+  const std::string session_id = response.session_snaphot().subscriber_id();

```

Comment:
session_id ? or subscriber_id?

Reply:
recently they changed to subscriber id. Its not a session_id.  session_id scope is limited to SMF, and it used in FEG

Reply:
Then should we do this instead? ` const std::string subscriber_id = response.session_snaphot().subscriber_id();`